Purchases/Public/Purchases.swift

```diff
@@ -797,7 +798,7 @@ public extension Purchases {
      * application agreements.
      * If you're having trouble see: https://www.revenuecat.com/2018/10/11/configuring-in-app-products-is-hard
      *
-     * - Parameter identifiers: A set of product identifiers for in app purchases setup via iTunesConnect.
+     * - Parameter identifiers: A set of product identifiers for in app purchases setup via AppStoreConnect.
```

Comment:
RIP iTunesConnect. You will not be missed.

Reply:
wonder if we should link to the ASC website.

Reply:
✅ 
Yeah, I think it's super easy to look up, but I don't have a problem adding it as well, so I did that